tests/test_filters.py

```diff
@@ -2375,6 +2375,19 @@ def test_extrude_rotate():
         and (ymax == line.bounds[1])
     )
 
+    rotation_axis = (0, 1, 0)
+    if pyvista.vtk_version_info >= (9, 1, 0):
+        poly = line.extrude_rotate(
+            rotation_axis=rotation_axis, resolution=resolution, progress_bar=True, capping=True
+        )
+        assert poly.n_cells == line.n_points - 1
+        assert poly.n_points == (resolution + 1) * line.n_points
+    elif:
```

Comment:
I am a Python beginner.
```suggestion
    else:
```